docs/detections/rules-ui-manage.asciidoc

```diff
@@ -121,18 +119,19 @@ IMPORTANT: Before manually running rules, make sure you properly understand and
 * Select all the rules you want to manually run, select the **Bulk actions** menu, then select **Manual run**.
 . Specify when the manual run starts and ends. The default selection is the current day starting three hours in the past. The rule will search for events during the selected time range.
 . Click **Run** to manually run the rule.
-+
-NOTE: Manual runs can produce multiple rule executions. This is determined by the manual run's time range and the rule's execution schedule.
 
-The manual run's details are shown in the <<manual-runs-table,Manual runs>> table on the *Execution results* tab. Changes you make to the manual run or rule settings will display in the Manual runs table after the current run completes.
+The rule runs over the time range that you selected. Note that all <<rule-notifications,alert actions>> for the rule are activated, except for custom **Summary of alerts** actions. 
```

Comment:
Please change to: " Note that all <<rule-notifications,alert actions>> for the rule are activated, except for actions for **Summary of alerts** at the custom frequency. "

Reply:
```suggestion
The rule runs over the time range that you selected. Note that all <<rule-notifications,alert actions>> for the rule are activated, except for **Summary of alerts** actions that run at a custom frequency.
```